source/appModules/explorer.py

```diff
@@ -267,10 +267,11 @@ def event_gainFocus(self, obj, nextHandler):
 			# Therefore, if there is a pending focus event, don't bother handling this event.
 			return
 
-		if wClass == "ForegroundStaging":
+		if wClass in ("ForegroundStaging", "LauncherTipWnd", "ApplicationManager_DesktopShellWindow"):
 			# #5116: The Windows 10 Task View fires foreground/focus on this weird invisible window before and after it appears.

```

Comment:
"This weird invisible window" needs clarification now. Please change this line to mention ForegroundStaging",